src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -20,6 +21,32 @@
         return filterMatchingPersons(configuration, t -> Association.closestToValue(t.getValue()) == association);
     }
 
+    /**
+     * Gives you a set of all your friends. No matter how friendly they are.
+     * The difference to {@link Operations#getByCloseToAssociation} is that you also get players with the
+     * friendliness value between excluding 0 and 0.5.
+     *
+     * @param configuration the config.
+     * @param <T>
+     * @return all players that are saved as your friends. No matter how friendly they are.
+     */
+    public static <T extends IPerson> Set<T> getAllFriends(final Configuration<T> configuration) {
+        return getInFriendlinessRange(configuration, Standard.STANDARD_NEUTRAL, Standard.STANDARD_MAX_FRIENDLINESS);
```

Comment:
`getInFriendlinessRange` is inclusive, meaning that this method will return persons with a value of `0` (`NEUTRAL`) as well. Maybe create an alternative of `getInFriendlinessRange`, but one that operates exclusively?

Reply:
Why are neutrals also your friends?
Wouldnt the same reasoning also mean that neutrals are you enemies?
Besides you cant get neutrals since we agreed to not store them?

Reply:
> Why are neutrals also your friends?
> Wouldnt the same reasoning also mean that neutrals are you enemies?

They're not, that's why I left this comment. `getAllFriends` shouldn't include neutral persons.

> Besides you cant get neutrals since we agreed to not store them?

I don't remember agreeing to this restriction. The standard doesn't impose this limitation - and for good reason.
What if we want to keep around some metadata for a player, e.g. their nickname or custom colour - this should be allowed to be in the friends list.

Reply:
oh shit. I just notice I did it wrong. sorry you are right.

`I don't remember agreeing to this restriction. The standard doesn't impose this limitation - and for good reason.
What if we want to keep around some metadata for a player, e.g. their nickname or custom colour - this should be allowed to be in the friends list.`
Yea I agree with that actually. Somebody else said that then. But it's good we can still count with that